site/content/contribute/server/feature-flags.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Feature Flags"
+date: 2020-10-15T16:00:00-0700
+weight: 3
+subsection: Server
+---
+
+
+Feature flags allow us to be more confident in shipping features continuously to Mattermost Cloud. Feature flags allow us to control which features are enabled on a cluster level.
+
+# How to use
+
+## Adding the feature flag in code
+
+1. Add the new flag to the feature flag struct located in `model/feature_flags.go`.
+2. Set a default value in the SetDefaults function in the same file.
+3. Use the feature flag in code as you would use a regular configuration setting. In tests, manipulate the configuration value to test value changes, such as activation and deactivation, of the feature flag.
+4. Code may be merged regardless of setup in the management system. In this case it will always take the default value supplied in the SetDefaults function.
```

Comment:
Nitx2:
```suggestion
4. Code may be merged regardless of setup in the management system. In this case it will always take the default value supplied in the `SetDefaults` function.
```